spec/features/viewing_a_manual_spec.rb

```diff
@@ -159,4 +159,38 @@
       expect(page.status_code).to eq(200)
     end
   end
+
+  describe "support for example manuals in govuk-content-schemas" do
+    it 'should render a vaguely correct page' do
```

Comment:
The description probably needs to be a bit less, well, vague. 'should render a page containing the title'?